packages/marko-web-algolia-site-search/components/pagination-controls.marko

```diff
@@ -0,0 +1,53 @@
+import { cleanPath } from "@parameter1/base-cms-utils";
```

Comment:
This could be included, but it should be a "functional" component only -- meaning, the site should determine how the controls are rendered (via a `input.renderBody` call) and the package component merely provides the wrapping logic